.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Master
+
+on:
+  push:
+    branches:
+      - master
+
+  # This allows us to manually run this job
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    if: github.repository == 'IABTechLab/uid2-android-sdk' && github.ref == 'refs/heads/master'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Upload Artifacts
+        uses: gradle/gradle-build-action@v2
```

Comment:
This means that every commit on `master` will be published to Maven. While `master` is being developed, this means that we'll push `-SNAPSHOT` builds. However, when we go through a release process, we change the version for one commit to be a non-snapshot version which will be fully published. Then the next commit moves it to the next version (snapshot).

This matches other popular SDKs, like OkHttp.

Reply:
Branch is `main`.